[c10d] Move pg wrapper tests to their own file.

rohan-varma opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#59840 [c10d] Move pg wrapper tests to their own file.**

moving these tests to their own standalone file. No meaningful code changes.

Differential Revision: [D29012664](https://our.internmc.facebook.com/intern/diff/D29012664/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D29012664/)!

rohan-varma committed:
[c10d] Move pg wrapper tests to their own file.

moving these tests to their own standalone file. No meaningful code changes.

Differential Revision: [D29012664](https://our.internmc.facebook.com/intern/diff/D29012664/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D29012664/)!

[ghstack-poisoned]

rohan-varma committed:
Update on "[c10d] Move pg wrapper tests to their own file."


moving these tests to their own standalone file. No meaningful code changes.

Differential Revision: [D29012664](https://our.internmc.facebook.com/intern/diff/D29012664/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D29012664/)!

[ghstack-poisoned]

cbalioglu reviewed (approved):
LGTM. Approving in advance. Once the flake8 issue is resolved, good to go.

rohan-varma committed:
Update on "[c10d] Move pg wrapper tests to their own file."


moving these tests to their own standalone file. No meaningful code changes.

Differential Revision: [D29012664](https://our.internmc.facebook.com/intern/diff/D29012664/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D29012664/)!

[ghstack-poisoned]

rohan-varma committed:
Update on "[c10d] Move pg wrapper tests to their own file."


moving these tests to their own standalone file. No meaningful code changes.

Differential Revision: [D29012664](https://our.internmc.facebook.com/intern/diff/D29012664/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D29012664/)!

[ghstack-poisoned]

rohan-varma committed:
Update on "[c10d] Move pg wrapper tests to their own file."


moving these tests to their own standalone file. No meaningful code changes.

Differential Revision: [D29012664](https://our.internmc.facebook.com/intern/diff/D29012664/)

**NOTE FOR REVIEWERS**: This PR has internal Facebook specific changes or comments, please review them on [Phabricator](https://our.internmc.facebook.com/intern/diff/D29012664/)!

[ghstack-poisoned]
